libs/types/src/election.ts

```diff
@@ -315,7 +294,7 @@ export const getPartyPrimaryAdjectiveFromBallotStyle = ({
   ballotStyleId: string
   election: Election
 }): string => {
-  const parts = ballotStyleId && ballotStyleId.match(/(\d+)(\w+)/i)
+  const parts = ballotStyleId && /(\d+)(\w+)/i.exec(ballotStyleId)
```

Comment:
This change was suggested by an eslint rule to prefer `exec` over `match` for non-`g` regexes.